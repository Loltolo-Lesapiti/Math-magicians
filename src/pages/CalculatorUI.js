import Calculator from "../components/Calculator";

const CalculatorUI = () => (
  <div className="container">
    <h2>Let&apos;s do same maths!</h2>
    <Calculator />
  </div>
);
export default CalculatorUI;
