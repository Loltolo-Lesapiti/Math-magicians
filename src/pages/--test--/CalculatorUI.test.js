import React from 'react';
import TestRenderer from 'react-test-renderer';
import CalculatorUI from '../CalculatorUI';
import '@testing-library/jest-dom';

describe('Calculator UI', () => {
  test('Testing Calculator Interface', () => {
    const tree = TestRenderer.create(<CalculatorUI />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
