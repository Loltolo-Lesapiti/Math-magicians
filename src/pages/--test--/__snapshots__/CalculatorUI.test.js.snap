// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator UI Testing Calculator Interface 1`] = `
<div
  className="calculatorContainer"
>
  <h2>
    Calculator
  </h2>
  <div
    className="cal-grid"
  >
    <button
      className="workspace"
      type="button"
    >
      0
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      %
    </button>
    <button
      className="cal-Btn operator"
      onClick={[Function]}
      type="button"
    >
      รท
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="cal-Btn operator"
      onClick={[Function]}
      type="button"
    >
      ร
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="cal-Btn operator"
      onClick={[Function]}
      type="button"
    >
      -
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="cal-Btn operator"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      className="cal-Btn zero"
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      className="cal-Btn"
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="cal-Btn operator"
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;
